### PR TITLE
English readme page

### PR DESCRIPTION
**Pull request by juliecbd**
English readme page

### CODE REVIEW

File: README.en.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Introduction to models
+
+PaddlePaddle provides a rich set of computing units to help you build a modular approach to the ever-changing learning model to solve different application problems. Here, we are for the common machine learning tasks, providing a different neural network model for everyone to learn and use.
```
Comment (jacquesqiao) [inline README.en.md]: `providing a different neural network model`
==>
`providing several neural network models` ?

**Review by wangkuiyi — APPROVED** [review]
LGTM.  Thanks!

I have https://github.com/juliecbd/models/pull/1 to make this PR passing CI test.

File: README.en.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+
+## 1. Word vector
```
Comment (jacquesqiao) [inline README.en.md]: seems  `Word Embedding` is more exact

Comment (jacquesqiao) [reply README.en.md]: https://scholar.google.com/scholar?q=word+embedding&btnG=&hl=zh-CN&as_sdt=0%2C5

**Pull request merged**